deepcell/utils/plot_utils.py

```diff
@@ -59,16 +59,19 @@ def get_js_video(images, batch=0, channel=0, cmap='jet', vmin=0, vmax=30):
         'cmap': cmap,
     }
 
+    if vmax == 0:
+        vmax = images.max()
```

Comment:
Why did this work before? Should we instead default `vmax=None`?

Reply:
It defaulted to a max of 30 - for some reason the docs had it as the vmax being number of frames? We could default it to None, doesn't really matter.